tutor/test_new_t1_14_TeacherReading.py

```diff
@@ -0,0 +1,2052 @@
+            )
+        )
+        assignment.select_sections(self.teacher.driver, ['1.1'])
+        self.teacher.find(
+            By.XPATH, '//button[text()="Add Readings"]'
+        ).click()
+
+        # Save as draft
+        self.teacher.wait.until(
+            expect.element_to_be_clickable(
+                (By.ID,'builder-draft-button')
+            )
+        ).click()
+
+        # Check if the draft is on the dashboard
+        try:
+            self.teacher.find(
+                By.XPATH,
+                '//label[contains(text(),"{0}")]'.format(assignment_name)
+            )
+        except NoSuchElementException:
+            self.teacher.find(
+                By.XPATH,
+                '//a[contains(@class,"header-control next")]'
+            ).click()
+            self.teacher.find(
+                By.XPATH,
+                '//label[contains(text(),"{0}")]'.format(assignment_name)
+            )
+
+        self.ps.test_updates['passed'] = True
+
+    # Case C132521 003 - Teacher | Create and publish a new unopened reading from calendar
+    @pytest.mark.skipif(str(132521) not in TESTS, reason='Excluded')
+    def test_teacher_create_and_publish_new_unopened_reading_from_calendar_132521(self):
+        """
+        Click on calendar date for desired due date
+        Click on the 'Add Reading' option
+        ***The teacher is taken to the Add Reading Assignment page***
+
+        Click on the info icon
+        ***Instructions about the Publish, Cancel, and Save As Draft statuses appear*** (T1.14.036)
+
+        Enter an assignment name into the Assignment name text box [user decision]
+        ***Typed text appears in the input box***
+
+        ['All Sections' should be selected by default]
+        Click on the Open Date text field and click on a date in the future in calendar pop-up
+        Click on the Due Date text field and click on a desired due date on calendar pop-up
+        Click on the "+ Add Readings" button
+        Click on section(s) to add to assignment
+        Scroll to bottom of the page
+        Click on the "Add Readings" button
+        Click on the "Publish" button
+
+        # Expected result
+        ***Takes user back to calendar dashboard. An opened assignment appears on user calendar dashboard on due date with correct readings***
+        """
+        # self.ps.test_'name'] = 't1.14.004' \
+        #                                + inspect.currentframe().f_code.co_name[4:]
+        # self.ps.test_updates['tags'] = ['t1', 't1.14', 't1.14.004', '7995']
+        self.ps.test_updates['passed'] = False
+
+        # Test steps and verification assertions
+        calendar_date = self.teacher.wait.until(
+            expect.element_to_be_clickable(
+                (By.XPATH, '//div[contains(@class,"Day--upcoming")]')
+            )
+        )
+        self.teacher.driver.execute_script(
+            'return arguments[0].scrollIntoView();', calendar_date)
```

Comment:
generally, when you have parameters on the following line, you should have one parameter per line